README.md

```diff
@@ -190,6 +203,61 @@ View demos:
 <button class="my-standout-button">Standout button</button>
 ```
 
+#### Icons
+[View demo](http://build.origami.ft.com/files/o-buttons/demos/icons.html)
```

Comment:
Also, you shouldn't be committing html files, so this should be pointing to the demos endpoint in the build service